4.0.2/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM debian:wheezy
+
+MAINTAINER Jo Shields <jo.shields@xamarin.com>
+
+#based on dockerfile by Michael Friis <friism@gmail.com>
+
+RUN apt-get update \
+	&& apt-get install -y curl \
+	&& rm -rf /var/lib/apt/lists/*
+
+RUN apt-key adv --keyserver pgp.mit.edu --recv-keys 3FA7E0328081BFF6A14DA29AA6A19B38D3D831EF
+
+RUN echo "deb http://download.mono-project.com/repo/debian wheezy/snapshots/4.0.2 main" > /etc/apt/sources.list.d/mono-xamarin.list \
```

Comment:
The 4.0.0 snapshot dir was actually updated with the 4.0.2.5 files, so in theory rebuilding the 4.0.0 image should work?